test/e2e/apimachinery/protocol.go

```diff
@@ -50,11 +50,13 @@ var _ = SIGDescribe("client-go should negotiate", func() {
 			cfg.AcceptContentTypes = accept
 
 			c := kubernetes.NewForConfigOrDie(cfg)
-			svcs, err := c.CoreV1().Services("default").Get(context.TODO(), "kubernetes", metav1.GetOptions{})
+			ns := f.Namespace.Name
+			before, err := c.CoreV1().ConfigMaps(ns).List(context.TODO(), metav1.ListOptions{})
 			framework.ExpectNoError(err)
-			rv, err := strconv.Atoi(svcs.ResourceVersion)
+			name := strings.NewReplacer("/", "-", ",", "-").Replace(s)
+			_, err = c.CoreV1().ConfigMaps(ns).Create(context.TODO(), &v1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: name}}, metav1.CreateOptions{})
```

Comment:
this is switching from service to configmaps... will it break on clusters that have processes that automatically create configmaps in new namespaces?

Reply:
I guess it tolerates any added/modified event... a process that automatically created *and then deleted* a config map in a brand new namespace could make this test fail, but that would be pretty weird

Reply:
That's possible - I can add a field selector on the name to the watch.